### PR TITLE
Fix/186/storage requirements

### PR DESCRIPTION
**Pull request by pattivacek**
* Improve description of Secondaries with limited storage.
* Move section on ECUs with filesystems to the ecus page.
* Add section for non-Uptane ECUs, i.e., ECUs without secondary storage.

Related to https://github.com/uptane/uptane-standard/issues/186 and https://github.com/uptane/uptane-standard/pull/203.

### CODE REVIEW

**Review by mnm678 — APPROVED** [review]

File: customizations.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The OEM SHOULD add two types of information used by the Director repository to t
...
 
-Finally, in order to install a delta image, an ECU SHOULD take one of the actions described in Table 1, depending on whether or not the delta image has been encrypted, and if the ECU has additional storage. Note that the OEM MAY use stream ciphers in order to enable on-the-fly decryption on ECUs that do not have additional storage. In this case, the ECU would decrypt the delta image as it is downloaded, then follow the remainder of the steps in the third box.
+Finally, in order to install a delta image, an ECU SHOULD take one of the actions described in Table 1, depending on whether or not the delta image has been encrypted, and if the ECU has enough additional storage to store a copy of the image. Note that the OEM MAY use stream ciphers in order to enable on-the-fly decryption on ECUs that do not have enough additional storage. In this case, the ECU would decrypt the delta image as it is downloaded, then follow the remainder of the steps in the third box.
```
Comment (jhdalek55) [inline customizations.md:30]:
```suggestion
Finally, in order to install a delta image, an ECU SHOULD take one of the actions described in Table 1, depending on whether or not the delta image has been encrypted, and if the ECU has sufficient additional storage to store a copy of the image. Note that the OEM MAY use stream ciphers in order to enable on-the-fly decryption on ECUs that do not have sufficient additional storage. In this case, the ECU would decrypt the delta image as it is downloaded, then follow the remainder of the steps in the third box.
```

**Review by jhdalek55 — CHANGES_REQUESTED** [review]
I made two small wording changes that I think make the meaning more explicit

**Comment by pattivacek** [comment]
> I made two small wording changes that I think make the meaning more explicit

Thanks! I've merged them in.

**Review by jhdalek55 — APPROVED** [review]

**Comment by jhdalek55** [comment]
I'm going to merge, OK?

**Comment by pattivacek** [comment]
> I'm going to merge, OK?

Gladly, thanks!

**Pull request merged**